regression/scripts/Vehicles/Tiltwing_CRM.py

```diff
@@ -0,0 +1,529 @@
+import SUAVE
+from SUAVE.Core import Units, Data
+import copy
+from SUAVE.Components.Energy.Networks.Vectored_Thrust import Vectored_Thrust
+from SUAVE.Methods.Power.Battery.Sizing import initialize_from_mass
+from SUAVE.Methods.Propulsion.electric_motor_sizing import size_from_mass , compute_optimal_motor_parameters
+from SUAVE.Methods.Propulsion import propeller_design 
+from SUAVE.Methods.Aerodynamics.Fidelity_Zero.Lift import compute_max_lift_coeff 
+from SUAVE.Methods.Weights.Buildups.Electric_Vectored_Thrust.empty import empty
+from SUAVE.Methods.Utilities.Chebyshev  import chebyshev_data
+
+import numpy as np
+import pylab as plt
+from copy import deepcopy
+
+
+
+def tw_vehicle_setup():
+    
+    # ------------------------------------------------------------------
+    #   Initialize the Vehicle
+    # ------------------------------------------------------------------    
+    
+    #vehicle = SUAVE.Vehicle(SUAVE.Input_Output.SUAVE.load('cora_vehicle.res'))
+    vehicle = SUAVE.Vehicle()
+    vehicle.tag = 'Vahana'
+    vehicle.configuration = 'eVTOL'
+    # ------------------------------------------------------------------
+    #   Vehicle-level Properties
+    # ------------------------------------------------------------------    
+    # mass properties
+    vehicle.mass_properties.takeoff         = 2250. * Units.lb 
+    vehicle.mass_properties.operating_empty = 2250. * Units.lb
+    vehicle.mass_properties.max_takeoff     = 2250. * Units.lb
+    vehicle.mass_properties.center_of_gravity = [ 2.0144,   0.  ,  0.]
+    
+     
+    # This needs updating
+    # basic parameters
+    vehicle.reference_area                    = 10.58275476  
+    vehicle.envelope.ultimate_load            = 5.7
+    vehicle.envelope.limit_load               = 3.    
+  
+    
+    
+    # ------------------------------------------------------				
+    # WINGS				
+    # ------------------------------------------------------				
+    # WING PROPERTIES	
+    wing = SUAVE.Components.Wings.Main_Wing()
+    wing.tag	                = 'main_wing'		
+    wing.aspect_ratio	        = 11.37706641		
+    wing.sweeps.quarter_chord	= 0.0
+    wing.thickness_to_chord	= 0.18		
+    wing.taper	                = 1.		
+    wing.span_efficiency	= 0.9		
+    wing.spans.projected	= 6.65	
+    wing.chords.root	        = 0.95	
+    wing.total_length	        = 0.95			
+    wing.chords.tip	        = 0.95	
+    wing.chords.mean_aerodynamic= 0.95			
+    wing.dihedral	        = 0.0		
+    wing.areas.reference	= 6.31		
+    wing.areas.wetted	        = 12.635		
+    wing.areas.exposed	        = 12.635		
+    wing.twists.root	        = 0.		
+    wing.twists.tip	        = 0.		
+    wing.origin	                = [0.0,  0.0 , 0.0]
+    wing.aerodynamic_center	= [0., 0., 0.]		   
+    wing.winglet_fraction       = 0.0  
+    wing.symmetric              = True
+    
+    # Segment 	
+    segment = SUAVE.Components.Wings.Segment()
+    segment.tag			= 'Section_1'		
```

Comment:
Clean up the data blocks in this file too.

Reply:
Changed to table format